packages/server/lib/controllers/sync.controller.ts

```diff
@@ -339,6 +338,8 @@ class SyncController {
     }
 
     public async triggerAction(req: Request, res: Response, next: NextFunction) {
+        const span = tracer.startSpan('sync.triggerAction');
```

Comment:
I think datadog instrumentation has already created a span, maybe you can fetch the active one:
`const span = tracer.scope().active()`. I believe @TBonnin already did something like this in persist

https://docs.datadoghq.com/tracing/guide/add_span_md_and_graph_it/